lib/node_modules/@stdlib/math/base/special/binomcoefln/benchmark/c/native/benchmark.c

```diff
@@ -91,17 +91,20 @@ static double rand_double( void ) {
 */
 static double benchmark( void ) {
 	double elapsed;
-	int64_t n;
-	int64_t k;
+	int64_t n[ 100 ];
+	int64_t k[ 100 ];
 	double y;
 	double t;
```

Comment:
```suggestion
	double y;
	double t;
	int64_t n[ 100 ];
	int64_t k[ 100 ];
```

Variables are listed in order of precision, with higher precision types first.